### ISSUE TITLE
Rename field in NGT Signbank

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/NGT-signbank#193]
The phonology field Location has been renamed "Placement Active Articulator RH" everywhere while working on the ASL Signbank; please change back to "Location" for NGT.

**Comment by Woseseltops** [comment]
Fixed! Not only that particular field - ALL American translations where visible instead of the default English ones. For example, the footer also said 'The ASL Signbank is licensed ...'. 

Interestingly, you unknowingly created the problem yourself :) . This is caused by the fact that Django always tries to show a translation, even if the region (in this case: en_US) doesn't match... only when it can find absolutely nothing, it will show the default. Because the default is what we want for the NGT Signbank, we simply don't generate an American .mo file for the NGT Signbank, so the American translations are not available. By running `python bin/develop.py compilemessages`, as specified at the wiki, you accidently created this file. The solution is to run `python compilemessages.py -l nl`. I've updated the instructions!
